### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by awharn**
Update the documentation to include better instructions for Linux and MacOS

Signed-off-by: Andrew W. Harn <andrew.harn@broadcom.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,13 @@ Before you install and use the plug-in:
...
     For connectivity to Db2 for z/OS, the Db2 region should be `db2connectactivated` or a `db2connect` license file should be provided. Copy the `db2connect` license file to the `[db2 plugin folder]/node_modules/ibm_db/installer/clidriver/license/` folder. For more information, see [Addressing the license requirement](https://docs.zowe.org/stable/user-guide/cli-db2plugin.html#addressing-the-license-requirement) on the Zowe Docs site. 
 
-- **(MacOS only)** Download and Install [Xcode](https://developer.apple.com/xcode/resources/).
+- **(Linux and MacOS only)** Download and install [node-gyp](https://www.npmjs.com/package/node-gyp) globally by running `npm install -g node-gyp`
+
+- **(MacOS only)** Download and Install [Xcode](https://developer.apple.com/xcode/resources/) and [Python 3](https://www.python.org/downloads/mac-osx/).
+
+- **(Linux only - Debian, Ubuntu)** Download and Install the following via apt: `python3 make g++`
+
+- **(Linux only - CentOS, RHEL)** Download and Install the following via yum: `python3 make gcc-c++`
```
Comment (zFernand0) [inline README.md:38]: I don't know if it's necessary to include a full `apt` or `yum` command.
I assume most users may be familiar with them.

**Review by zFernand0 — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (awharn) [reply README.md:38]: We can include it if you prefer. Might be more clear, and you wouldn't need to think about it.
I wish I could just make this one line, but the g++ vs gcc-c++ package name difference ruins that idea. ☹️

**Comment by awharn** [comment]
I recommend this PR be merged before #84 if possible, so the new documentation gets included on NPM. 😄

**Review by t1m0thyj — APPROVED** [review]
LGTM!

In a CentOS 7 Docker container, installing the documented packages `python3 make gcc-c++` fixed the error on ibm_db install 🙂 In my case I didn't need to install node-gyp, but it's probably good to have that documented in case some users have an old version that relied on Python 2.

File: README.md (hunk 1 above)
Comment (zFernand0) [reply README.md:38]: Nah, I think it's fine as it is 😋

**Comment by zFernand0** [comment]
agent will be updated in subsequent PR 😋

**Pull request merged**